src/store/index.ts

```diff
@@ -286,6 +287,16 @@ export default createStore({
                 );
         },
 
+        ADD_PROJECT(state, project: Project) {
+            db.collection("projects")
+                .doc(project.id)
+                .set(project).then(() => {
+                    state.projects.push(project)
+                }).catch(function(error) {
+                    console.log("Error getting document: " + error)
```

Comment:
Try not to directly print out error messages for security reasons. try logging `Error getting document`or remove console.log